AutomatedLabWorker/AutomatedLabVMWareWorkerVirtualMachines.psm1

```diff
@@ -1,11 +1,53 @@
+<# 
+    An AutomatedLab user could have a VMWare environment on premisis _and_ have Hyper-V enabled on his own device.
+    When the user has both Hyper-V and VMWare modules loaded in one session, this can cause unwanted behaviour.
+    
+    This may need to be mitigated in AutomatedLab in some cases. There are two main approaches:
+    1) Prepending conflicting CmdLets with the name of the intende module of origin, e.g.:
+        VMware.VimAutomatiion.Core\Get-VM 
```

Comment:
I went that way, and added VMware.VimAutomation.Core\ before every CmdLet that conflicts between Hyper-V and VMware.VimAutomation.Core.